spec/controllers/api/v1/channels_controller_spec.rb

```diff
@@ -0,0 +1,66 @@
+require 'rails_helper'
+
+RSpec.describe Api::V1::ChannelsController, type: :controller do
+  let!(:organization) { create(:organization) }
+  let!(:channel) { create(:channel, organization: organization) }
+
+  describe 'GET #show' do
+    it 'return channel' do
+      request.headers['token'] = organization.token
```

Comment:
make some test helper, that's potential problem carrier
see, u have updated the api controller to fetch `headers['Token']` but within tests u have `headers['token']`

makes it wonder if tests are working and reliability of such tests